setup.py

```diff
@@ -57,10 +57,13 @@ def multiglob(*globs):
     ('share/applications', multiglob('*.desktop')),
     ('share/metainfo', multiglob('*.appdata.xml')),
     ('share/pronterface', multiglob('images/*.png',
-                                    'images/*.svg',
-                                    'locale/*/LC_MESSAGES/*.mo')),
+                                    'images/*.svg')),
 ]
 
+for locale in glob.glob('locale/*/LC_MESSAGES/'):
+    data_files.append((f'share/{locale}', glob.glob(f'{locale}/*.mo')))
```

Comment:
I think that `locale` folder should be inside `pronterface`?

`f'share/pronterface/{locale}'`

Reply:
This was the intention originally, but the locales belong to a shared location. Examine your /usr/share/locale.

Reply:
Got it, thanks for explanation.

Reply:
I've adapted the code to expect this.